ocis/pkg/runtime/service/service.go

```diff
@@ -486,6 +488,8 @@ func (s *Service) List(_ struct{}, reply *string) error {
 	table := tablewriter.NewWriter(tableString)
 	table.SetHeader([]string{"Service"})
 
+	s.mu.Lock()
+	defer s.mu.Unlock()
```

Comment:
something to consider is to move the unlock right after the `for` so the mutex is locked the least amount of time. In this case it isn't a big deal because the function shouldn't take a lot of time to complete, so the mutex should be locked for a little time.